setup.py

```diff
@@ -175,7 +175,7 @@ def on_exit():
         define_macros=macros)
     extensions = [ext, posix_extension]
 # Solaris
-elif sys.platform.lower().startswith('sunos'):
+elif sys.platform.lower().startswith('sunos') or sys.platform.lower().startswith('solaris'):
```

Comment:
Unfortunately my solaris build machines do not have mock on them and they fail to run the tests. I'll see what I can do about it.